jax/interpreters/pxla.py

```diff
@@ -639,10 +714,12 @@ def replicate(val, axis_size, nrep, devices=None, backend=None):
     devices = xb.get_backend(backend).get_default_device_assignment(nrep)
   assert nrep == len(devices)
 
+  # TODO(skye): use ShardingSpec.replication_factor instead
```

Comment:
Don't worry about this; I'll do it as part of the pmap-with-replication changes.

Reply:
Changed to TODO(jekbradbury) :)